src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/Microsoft.NET.Sdk.WebAssembly.Pack.pkgproj

```diff
@@ -7,7 +7,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(RepoTasksDir)Microsoft.NET.Sdk.WebAssembly.Pack.Tasks\Microsoft.NET.Sdk.WebAssembly.Pack.Tasks.csproj" />
-    <PackageFile Include="build\*.props;build\*.targets;build\*.web.config" TargetPath="build" />
+    <PackageFile Include="Sdk\*.props;Sdk\*.targets;Sdk\*.web.config" TargetPath="Sdk" />
```

Comment:
This didn't worked for me. How is it that it works now? Will it still work in blazor without workload?